settings/settings.json

```
@@ -31,7 +31,20 @@
         "description": "Settings relating to how the application looks.",
         "children": [
             {
-                "title": "Details Page",
+                "title": "Home Page",
+                "description": "Options for Home page.",
+                "children": [
+                    {
+                        "title": "Max Days Next Up",
+                        "description": "Set the maximum amount of days a show should stay in the 'Next Up' list without watching it.",
+                        "settingName": "ui.details.maxdaysnextup",
+                        "type": "integer",
+                        "default": "365"
+                    }
+                ]
+            },
+            {
+                "title": "Detail Page",
```

Corrected line was deleted above and then re-added with the old name
```suggestion
                "title": "Details Page",
```

---

This was one of the conflicts I resolved.